Topics/Development_Process/App_Store_Deployment.md

```diff
@@ -0,0 +1,158 @@
+# Apple App Store Deployment
+
+## Table of Contents
+### [Introduction](#introduction-1)
+### [Requirements](#requirements-1)
+### [iOS Related Certificates](#ios-related-certificates-1)
+### [Certificate Creation](#certificate-creation-1)
+### [Beta Testing](#beta-testing-1)
+### [TestFlight](#testflight-1)
+### [Ad hoc](#ad-hoc-1)
+### [Distribution Process](#distribution-process-1)
+### [App Store Submission](#app-store-submission-1)
+### [Additional Information](#additional-information-1)
+
```

Comment:
I was more considering putting the table of contents overall, and then a separate table of contents for the topics under beta testing, however, if you would like to do so like this it is fine, but to make this better I would suggest using bullet points for the topics that are sub-subtopics under the topics they are children of.

Reply:
updated to include this